tensorflow_addons/custom_ops/image/cc/kernels/euclidean_distance_transform_op.cc

```diff
@@ -77,17 +77,18 @@ TF_CALL_double(REGISTER);
 
 #undef REGISTER
 
-#if GOOGLE_CUDA
+// TODO: fix compile issue of the gpu kernel.
+#if 0 && GOOGLE_CUDA
```

Comment:
@mels630 Matt, seems that we can migrate gpu kernel after the PR is merged, right?

Reply:
no objection to fixing in a later PR. As long as we can avoid losing track of the issue!

Reply:
sure, file https://github.com/tensorflow/addons/issues/349 issue